openshift-with-appstudio-test/e2e/jvm_build_service_test.go

```diff
@@ -404,12 +404,14 @@ func TestExampleRun(t *testing.T) {
 					break
 				}
 			}
-			dbComplete := false
 			dbList, err := jvmClient.JvmbuildserviceV1alpha1().DependencyBuilds(ta.ns).List(context.TODO(), metav1.ListOptions{})
 			ta.Logf(fmt.Sprintf("number of dependencybuilds: %d", len(dbList.Items)))
+			dbComplete := true
 			for _, db := range dbList.Items {
-				if db.Status.State == v1alpha1.DependencyBuildStateComplete {
-					dbComplete = true
+				if db.Status.State != v1alpha1.DependencyBuildStateComplete {
```

Comment:
cool deal @dwalluck 

let's see how the in flight e2e test does

@stuartwdouglas FYI